doc/sphinx/source/quickstart/installation.rst

```diff
@@ -69,8 +69,96 @@ Note that the ESMValTool source code is contained in the ``esmvaltool-python`` p
 Docker installation
 ===================
 
-.. warning::
-    Docker section to be added
+ESMValTool is also provided through `DockerHub <https://hub.docker.com/u/esmvalgroup/>`_
+in the form of docker containers.
+See https://docs.docker.com for more information about docker containers and how to
+run them.
+
+You can get the latest release with
+
+.. code-block:: bash
+
+   docker pull esmvalgroup/esmvaltool:stable
+
+If you want to use the current master branch, use
+
+.. code-block:: bash
+
+   docker pull esmvalgroup/esmvaltool:latest
+
+To run a container using those images, use:
+
+.. code-block:: bash
+
+   docker run esmvalgroup/esmvaltool:stable --help
+
+Note that the container does not see the data or environmental variables
+available in the host by default. You can make data available with
+``-v /path:/path/in/container`` and environmental variables with ``-e VARNAME``.
+
+For example, the following command would run a recipe
+
+.. code-block:: bash
+
+   docker run -e HOME -v "$HOME":"$HOME" -v /data:/data esmvalgroup/esmvaltool:stable -c ~/config-user.yml ~/recipes/recipe_example.yml
```

Comment:
All the commands need to be updated according to the new command line interface. It would also be nice to show how to list the available recipes and run an installed recipe instead of a recipe from the home directory.